op-challenger/flags/flags.go

```diff
@@ -84,6 +84,11 @@ var (
 		Usage:   "L2 Address of L2 JSON-RPC endpoint to use (eth and debug namespace required)  (cannon/asterisc trace type only)",
 		EnvVars: prefixEnvVars("L2_ETH_RPC"),
 	}
+	L2ExperimentalEthRpcFlag = &cli.StringFlag{
```

Comment:
these changes will be rebased away here: https://github.com/ethereum-optimism/optimism/pull/12559